### PR TITLE
Added April, renamed release notes for management, redid index and tool to match

### PR DESCRIPTION
**Pull request by djw-m**
Signed-off-by: Dj Walker-Morgan <dj.walker-morgan@enterprisedb.com>

## What Changed?

Added April, 
renamed release notes for better management, 
redid index and tool to match

### CODE REVIEW

File: tools/user/import/bareleasenotes/barelease.js [hunk 1]
```diff
@@ -95,9 +95,12 @@ async function fetchAndProcess(directory, currentYear, currentMonth) {
     const releaseNotesBody = cleanLines
       .map((line) => `| Enhancement | ${line.trim()} |`)
       .join("\n");
-    const releaseNotesFileName = `${directory}/${getShortMonthName(
-      currentMonth,
-    )}_${currentYear}_rel_notes.mdx`;
+    // file name shoudl be like 2021_09_sep_rel_notes.mdx
```
Comment (gvasquezvargas) [inline tools/user/import/bareleasenotes/barelease.js:98]: just a nit
```suggestion
    // file name should follow the pattern 2021_09_sep_rel_notes.mdx
```

**Review by gvasquezvargas — COMMENTED** [review]
Two minor comments

File: product_docs/docs/biganimal/release/release_notes/2024_04_apr_rel_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: BigAnimal April 2024 release notes
+navTitle: April 2024
+---
+
+BigAnimal's April 2024 includes the following enhancements and bugfixes:
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/biganimal/release/release_notes/2024_04_apr_rel_notes.mdx:6]: Only there isn't an actual release, just rolling features being added.

```suggestion
In April 2024 BigAnimal saw the following enhancements and bugfixes:
```

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**